include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -7454,37 +7435,29 @@ namespace internal
   /**
    * Implementation for vectors that have the begin() methods.
    */
-  template <int dim,
-            int fe_degree,
-            int n_q_points_1d,
-            typename Number,
+  template <typename Number,
             typename VectorizedArrayType,
             typename VectorType,
+            typename T,
```

Comment:
Could you be explicit here? Says the person who is probably responsible for the most `T` in the matrix_free folder^^

Reply:
OK